### PR TITLE
Refresh the Primer with updated images

### PR DESCRIPTION
**Pull request by EdwardChaplinArchilizer**
### Purpose

This PR replaces outdated images in the primer with updated visuals to reflect the latest state of dynamo, the blog, forum and github. All changes have been made with the styling of the previous images taken into account, so the same colour for highlights has been used and images have been taken in a way to best replicate the image they are replacing. 
These updated images should improve the clarity and user experience for customers using the Primer. 

Key additions:
Re-created images with the old python UI to have the same content but with the new python UI
Replaced images where removed nodes were present (e.g the Formula Node)
Updated images of the blog/github/forum where noticeable changes were present
Updated images of the dynamo UI and menus

Aimed to keep the new images the same width and height as the ones they replaced, though this was hard for a few that had very large dimensions originally. Example: 9_best_practices/images/2/thinkparametrically02.jpg

### Declarations
Check these if you believe they are true
- [ ]  The codebase is in a better state after this PR
- [ ]  Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ]  The level of testing this PR includes is appropriate
- [ ]  User facing strings, if any, are extracted into *.resx files
- [ ]  All tests pass using the self-service CI.
- [ ]  Snapshot of UI changes, if any.
- [ ]  Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ]  This PR modifies some build requirements and the readme is updated
- [ ]  This PR contains no files larger than 50 MB

### Release Notes
Updating Images across the primer to replace any that could cause confusion to customers.

Reviewers
@dnenov
@achintyabhat
@QilongTang

Informed
@Amoursol

### CODE REVIEW

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
Looks way nicer than the outdated images, three of them from Chapter 9 are now much smaller but I think that's OK because the previous image size are too large. I am going to merge for now and check locally later about chapter 9.

**Comment by EdwardChaplinArchilizer** [comment]
> Looks way nicer than the outdated images, three of them from Chapter 9 are now much smaller but I think that's OK because the previous image size are too large. I am going to merge for now and check locally later about chapter 9.

Yes there were some very large images I had to replace, I don't think my monitors are big enough to replicate the size easily but looking at section 9 I think the images look good with no need for changing. What do you think?

**Comment by QilongTang** [comment]
> > Looks way nicer than the outdated images, three of them from Chapter 9 are now much smaller but I think that's OK because the previous image size are too large. I am going to merge for now and check locally later about chapter 9.
> 
> Yes there were some very large images I had to replace, I don't think my monitors are big enough to replicate the size easily but looking at section 9 I think the images look good with no need for changing. What do you think?

Thank you for the background, I checked locally and it looks good! No need to update again

File: 7_dynamo_for_revit/images/5/customizing-exercise06.jpg
Comment (achintyabhat) [inline 7_dynamo_for_revit/images/5/customizing-exercise06.jpg:1]: @EdwardChaplinArchilizer There have been more improvements to package manager in 3.4. Can you please update the screenshot here?

**Pull request merged**